app/components/application-wrapper.js

```diff
@@ -188,8 +188,8 @@ export default Component.extend({
 			 * pass it up to handleLink
 			 */
 			if (!target.href.match(`^${window.location.origin}/a/.*/comments$`)) {
-				this.sendAction('closeLightbox');
-				this.sendAction('handleLink', target);
+				this.get('closeLightbox')();
```

Comment:
Don't we loose `this` context is such execution? Or ember binds it correctly?

Reply:
nope, Ember handles it like it should
https://ember-twiddle.com/55e68273e4e681d56a131b88f11cca3e?openFiles=controllers.application.js%2Ctemplates.components.my-component.hbs